ubuntu-18.04/upgrade_server

```diff
@@ -25,6 +28,13 @@ function main() {
   #############################################################################
 
   logger::action "Updating server package index files..."
+  # Fix for issue-881.
+  waiting_time=0
+  while [[ $(pgrep apt) && ${waiting_time} -lt ${MAX_WAIT_TIME} ]]; do
+    ((waiting_time++))
+    logger::info "Waiting for other apt commands to finish: ${waiting_time} sec / ${MAX_WAIT_TIME} sec...";
+    sleep 1;
+  done;
```

Comment:
Je pense quon devrait ajouter de quoi de semblable a sa:

if [[ $(pgrep apt) ]]; then
  logger::error "apt still running after 60 s, stopping the script";
  exit;
fi

au lieu de continuer dans le script si sa fait 60 s et que apt roule encore, au moin le script va arreter